docs/quickstart.md

```diff
@@ -97,7 +97,7 @@ If you want, you can show a loading dialog before docsify starts to render your
 <div id="app">Please wait...</div>
 ```
 
-You should set the `data-app` attribute if you changed `el`:
+By default, the `id` attribute of the main container is `app`. If you want to use a different id, you must also set the `data-app` attribute, and [configure](configuration.md#el) the `el` parameter accordingly:
```

Comment:
A bit weird that we need to set `data-app` like this. Let's fix in v5.

/wish-there-was-a-bot-command-to-open-an-new-issue-right-from-here

Reply:
I added it manually https://github.com/docsifyjs/docsify/projects/2#card-51972364

Reply:
@sy-records let's get things moving. Let's make changes more organic (as per the other v5 topic I closed), and we can release a major version bump any time we fix a small feature that is breaking. In this case, this one is worth it in my opinion.